app/static/src/app/components/options/NumericInput.vue

```diff
@@ -69,10 +74,74 @@ export default defineComponent({
             }
 
             if (blur) {
-                errorTooltipProps.content = "";
+                // wrap tooltip props update with update to input box
+                // as the v-tooltip directive needs an update to actually
+                // hide the tooltip
+                textValue.value = `${textValue.value} `;
+                tooltipProps.value.content = "";
+                textValue.value = textValue.value.trim();
```

Comment:
Not sure this is the right place to comment this, and this does not need fixing here, but it would be nice if the tooltip appeared when the cursor is active in the control if it _should_ be displayed.

So if you enter a value of 70 it shows you the "traces will be hidden" value as expected. You hit "Run model", then click back in the control but nothing is displayed - if you then change the value (say to 700 by typing 0) the tip reappears but it might be nice if it was visible immediately when the cursor was active

Reply:
yep can do that pretty easily actually! these are the exact correct lines to comment this, will involve this method of doing a dummy update when the user focuses on the component! ticket: https://mrc-ide.myjetbrains.com/youtrack/agiles/103-57/current?issue=mrc-4605